<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoToOffsetDialog</class>
 <widget class="QDialog" name="GoToOffsetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Offset Jump</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>481</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>From Start</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="fromCursorRadioButton">
         <property name="text">
          <string>From Cursor Position</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QRadioButton" name="fromEndRadioButton">
         <property name="text">
          <string>From End</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="offsetLineEdit">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="formatComboBox">
        <item>
         <property name="text">
          <string>Decimal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hex</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Multiplier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="multiplierComboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>512</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2048</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4096</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8192</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="clusterJumpTab">
    <attribute name="title">
     <string>Cluster Jump</string>
    </attribute>
    <widget class="QComboBox" name="partitionsComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="clusterNumberlineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>enter cluster number</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="jumpButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Jump</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
